### PR TITLE
microverse Project

### PR DESCRIPTION
**Pull request by Haadiiii**
## MIcroverse : Hello World project

in this project we:
-Added an HTML file with text.
-Added some styles
-Added a microverse template README file.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,2 @@
+
+node_modules
```
Comment (omarsalem7) [inline .gitignore:2]: - [ ] Please add `test.md` file in `.gitignore`

File: README.md [hunk 2]
```diff
@@ -1,73 +1,48 @@
...
-
-## 📝 License
-
-This project is [MIT](./MIT.md) licensed.
+# Hello microverse project
+
+# description
+
+1. Added `Project.css` as an external style sheet to the `Preoject.html` file.
+
```
Comment (omarsalem7) [inline README.md:6]: - [ ] Kindly use [this template] and customize it for your project and remove unneeded sections.

**Review by omarsalem7 — CHANGES_REQUESTED** [review]
Hi @Haadiiii 😊, 

Good job so far! 👏
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights
- Descriptive pull request ✔️
- No linter errors ✔️
- Descriptive commit messages ✔️

## Required Changes ♻️

- [ ] Kindly make sure your Pull Request should have a summary describing introduced changes in more detail.
- [ ]  _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Kossi-stack — APPROVED** [review]
## Status: Approved ✔️

**Hi @Haadiiii,**

You did a really great job! 💯 💯💯💯
Congratulations! 🎉 🎉 🎉 🎉 

128382030-38e7de91-f6f2-45cf-b54e-8068e0446e4c

You may close this Pull Request and merge the changes
Keep being awesome dev's and good luck with your next milestone!

## Optional suggestions
 - [ ] Kindly add a descriptive pull request message, This helps other programmers who want to take a look at your code to understand what changes you made to a particular branch, you can find an example [here](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md#5-basic-rules-to-follow-in-microverse-projects).
 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏 👏 👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**